src/main/java/io/github/genomicdatainfrastructure/discovery/repositories/CkanDatasetsRepository.java

```diff
@@ -32,10 +32,17 @@ public CkanDatasetsRepository(
     public DatasetsSearchResponse search(DatasetSearchQuery query, String accessToken) {
         var facetsQuery = CkanFacetsQueryBuilder.buildFacetQuery(query);
 
+        String sort_string = query.getSort();
+        if (sort_string != null) {
+            if (sort_string.contains("title") && !sort_string.contains("title_string")) {
+                sort_string = sort_string.replace("title", "title_string");
+            }
+        }
+
         var response = ckanQueryApi.packageSearch(
                 query.getQuery(),
                 facetsQuery,
-                query.getSort(),
+                sort_string,
```

Comment:
here also. camel case instead of using an underscore

Reply:
Done